client/my-sites/plugins-wpcom/plugin-install-button.jsx

```diff
@@ -37,13 +38,23 @@ export const WpcomPluginInstallButton = props => {
 	function installButtonAction( event ) {
 		event.preventDefault();
 
-		const hasErrors = !! get( eligibilityData, 'eligibilityHolds', [] ).length;
-		const hasWarnings = !! get( eligibilityData, 'eligibilityWarnings', [] ).length;
+		const eligibilityHolds = get( eligibilityData, 'eligibilityHolds', [] );
+		const eligibilityWarnings = get( eligibilityData, 'eligibilityWarnings', [] );
+
+		const hasErrors = !! eligibilityHolds.length;
+		const hasWarnings = !! eligibilityWarnings.length;
 
 		if ( ! hasErrors && ! hasWarnings ) {
 			// No need to show eligibility warnings page, initiate transfer immediately
 			initiateTransfer( siteId, null, plugin.slug );
 		} else {
+			analytics.tracks.recordEvent( 'calypso_automatic_transfer_plugin_install_ineligible',
```

Comment:
So this can become
```
import { recordTracksEvent } from 'client/state/analytics/actions';
//...
this.props.recordTracksEvent( 'calypso_automatic_transfer_plugin_install_ineligible', analyticsProps } );
//...
const mapDispatchToProps = {
    initiateTransfer: initiateThemeTransfer,
    recordTracksEvent,
};